_includes/api/en/4x/router-instance.md

```diff
@@ -0,0 +1,43 @@
+<h3 id='router.instance'>router(req, res, next)</h3>
+
+Once a router instance has been created and all the relevant route handlers have been added
+you can use in a few different ways.
+
+```js
+const userRouter = express.Router()
+
+userRouter.get('/events', (req, res, next) => {
+  const { userId } = req.query
+  // return user specific events
+})
+
+userRouter.get('/tasks', (req, res, next) => {
+  const { userId } = req.query
+  // return user specific tasks
+})
+```
+
+  - Directly pass it to another router's route handler.
+
+```js
+app.use('/calendar', userRouter)
+```
+
+  - Or you can call pass the `req`, `res`, and `next` arguments to the `Router` instance as a function
+  within another router's route handler.
```

Comment:
This wording is very confusing to me...  It says
"...passing req, res, and next arguments to the `Router` instance as a function....",  but AFAICT the function is being passed to an `app` instance, not a `Router` instance.  

Maybe try rewording into multiple sentences ..?